src/views/tutorial-view/index.tsx

```diff
@@ -74,48 +77,60 @@ export default function TutorialView({
     video,
     collectionCtx,
   } = tutorial
+
+  const InteractiveLabWrapper = handsOnLab?.id ? InstruqtProvider : Fragment
```

Comment:
Relevant to previous comment, I think you could define a `isInteractive` boolean variable above this and use it on this line and down below in the markup where my other comment is.